tests/lean/625.lean

```diff
@@ -1,8 +1,8 @@
 import Lean
 open Lean Lean.PrettyPrinter
 
-def pfoo : PUnit → PUnit := id
-def px   : PUnit := ()
+def foo : PUnit → PUnit := id
+def x   : PUnit := ()
```

Comment:
I believe this was the original intent behind this test (at least as far as I can tell from the comment).